install.sh

```diff
@@ -574,6 +585,21 @@ do_install_tar() {
     fi
 }
 
+setup_fapolicy_rules() {
+    if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ] || [ -r /etc/rocky-release ]; then
+        verify_fapolicyd || return
+        # setting rke2 fapolicyd rules
+        cat <<-EOF >>"/etc/fapolicyd/rules.d/80-rke2.rules"
+allow perm=any all : dir=/var/lib/rancher/
+allow perm=any all : dir=/opt/cni/
+allow perm=any all : dir=/run/k3s/
+allow perm=any all : dir=/var/lib/kubelet/
+EOF
+        fagenrules --load || fatal "failed to load rke2 fapolicyd rules"
+        systemctl restart fapolicyd
```

Comment:
We should probably honor `INSTALL_RKE2_SKIP_RELOAD` and not try to make any changes to the running system if it's set - see https://github.com/rancher/rke2/pull/3248

Reply:
will fix